### ISSUE TITLE
ActionView::MissingTemplate: Missing template homepage/index, application/index with {:locale=>[:es], :formats=>[:atom], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :coffee, :jbuilder]}. Searched in:
  * "/aytomad/app/agendas/agendas/releases/

### ISSUE THREAD

**Issue by voodoorai2000** [AyuntamientoMadrid/agendas#266]
View details in Rollbar: [https://rollbar.com/consul/Agendas/items/12/](https://rollbar.com/consul/Agendas/items/12/)

```

ActionView::MissingTemplate: Missing template homepage/index, application/index with {:locale=>[:es], :formats=>[:atom], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :coffee, :jbuilder]}. Searched in:
  * "/aytomad/app/agendas/agendas/releases/20171229110622/app/views"
  * "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/kaminari-core-1.0.1/app/views"
  * "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/foundation-rails-5.5.1.0/app/views"
  * "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/devise-3.5.10/app/views"

  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/path_set.rb", line 46, in find
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/lookup_context.rb", line 121, in find
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/abstract_renderer.rb", line 18, in find_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/template_renderer.rb", line 40, in determine_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/template_renderer.rb", line 8, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/renderer.rb", line 46, in render_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/renderer/renderer.rb", line 27, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/rendering.rb", line 100, in _render_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/streaming.rb", line 217, in _render_template
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionview-4.2.10/lib/action_view/rendering.rb", line 83, in render_to_body
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/rendering.rb", line 32, in render_to_body
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/renderers.rb", line 37, in render_to_body
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/abstract_controller/rendering.rb", line 25, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/rendering.rb", line 16, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/instrumentation.rb", line 44, in block (2 levels) in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/core_ext/benchmark.rb", line 12, in block in ms
  File "/aytomad/app/agendas/.rvm/rubies/ruby-2.2.1/lib/ruby/2.2.0/benchmark.rb", line 303, in realtime
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/core_ext/benchmark.rb", line 12, in ms
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/instrumentation.rb", line 44, in block in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/instrumentation.rb", line 87, in cleanup_view_runtime
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activerecord-4.2.10/lib/active_record/railties/controller_runtime.rb", line 25, in cleanup_view_runtime
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/sunspot_rails-2.2.0/lib/sunspot/rails/railties/controller_runtime.rb", line 15, in cleanup_view_runtime
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/instrumentation.rb", line 43, in render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/implicit_render.rb", line 10, in default_render
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/implicit_render.rb", line 5, in send_action
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/abstract_controller/base.rb", line 198, in process_action
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/rendering.rb", line 10, in process_action
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/abstract_controller/callbacks.rb", line 20, in block in process_action
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/callbacks.rb", line 117, in call
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/callbacks.rb", line 117, in call
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/callbacks.rb", line 555, in block (2 levels) in compile
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/callbacks.rb", line 505, in call
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/callbacks.rb", line 505, in call
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/callbacks.rb", line 92, in __run_callbacks__
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/callbacks.rb", line 778, in _run_process_action_callbacks
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/callbacks.rb", line 81, in run_callbacks
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/abstract_controller/callbacks.rb", line 19, in process_action
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/rescue.rb", line 29, in process_action
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/actionpack-4.2.10/lib/action_controller/metal/instrumentation.rb", line 32, in block in process_action
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications.rb", line 164, in block in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2.0/gems/activesupport-4.2.10/lib/active_support/notifications/instrumenter.rb", line 20, in instrument
  File "/aytomad/app/agendas/agendas/shared/bundle/ruby/2.2

**Comment by Senen** [comment]
Error produced during production deployment!

**Comment by raul-fuentes** [comment]
I'll do this one

**Comment by Senen** [comment]
Closed via #316
